Task6/index.html

```diff
@@ -0,0 +1,156 @@
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-5"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-6"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-7"></div>
+        </div>
+
+        <div class="ts6-social-line">
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-1"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-2"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-3"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-4"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-5"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-6"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-7"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-8"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-9"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-10"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-11"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-1"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-2"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-3"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-4"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-5"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-6"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-7"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-4"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-5"></div>
+            <div class="ts6-social-line__icon ts6-social-line__icon_icon-6"></div>
+        </div>
+
+
+    </div>
+    <div class="ts6-signboard">Social Sprites Icons</div>
+</header>
+
+<main>
+
+  <div class="ts6-layout">
+      <div class="ts6-field">
+            <button class="ts6-button__field ts6-button-large ts6-pink">Pink Button</button>
+            <button class="ts6-button__field ts6-button-middle ts6-green">Green Button</button>
+      </div>
+
+      <div class="ts6-field">
+            <button class="ts6-button__field ts6-button-middle ts6-blue">Blue Button</button>
+            <button class="ts6-button__field ts6-button-middle ts6-red">Red Button</button>
+      </div>
+
+      <div class="ts6-field">
+            <button class="ts6-button__field ts6-button-middle ts6-orange">Orange Button</button>
+            <button class="ts6-button__field ts6-button-middle ts6-yellow">Yellow Button</button>
+      </div>
+  </div>
+
+  <div class="ts6-layout ts6-dark">
```

Comment:
`ts6-dark` - Looks like it should be a modifier for `ts6-layout`. Isn't it?
Also, in my opinion class name `layout` doesn't reflect a semantic meaning that this element contains.

Reply:
Done.